plugins/modules/inventory_intent.py

```diff
@@ -2567,6 +2567,37 @@ def get_diff_merged(self, config):
                             self.msg = "Device with IP address '{0}' already exists in inventory".format(new_mgmt_ipaddress)
                             self.log(self.msg, "ERROR")
                             self.result['response'] = self.msg
+                        else:
+                            response = self.dnac._exec(
+                                family="devices",
+                                function='sync_devices',
+                                op_modifies=True,
+                                params=playbook_params,
+                            )
+                            self.log("Received API response from 'sync_devices': {0}".format(str(response)), "DEBUG")
+
+                            if response and isinstance(response, dict):
+                                task_id = response.get('response').get('taskId')
+
+                                while True:
+                                    execution_details = self.get_task_details(task_id)
+                                    if execution_details.get("isError"):
+                                        self.status = "failed"
+                                        failure_reason = execution_details.get("failureReason")
+                                        if failure_reason:
+                                            self.msg = "Device Updation get failed because of {0}".format(failure_reason)
```

Comment:
Failed to update device due to {0}".format(failure_reason) Also specify which device .. may be device ip

Reply:
added the log message with device ip.